src/css/curCSS.js

```diff
@@ -16,7 +16,10 @@ function curCSS( elem, name, computed ) {
 	computed = computed || getStyles( elem );
 
 	// Support: IE <=9 only
-	// getPropertyValue is only needed for .css('filter') (#12537)
+	//
+	// getPropertyValue is needed for:
+	//   .css('filter') (#12537)
+	//   .css('--customProperty) (#3144)
```

Comment:
Maybe the support comment should be removed in favor of `//   .css('filter') in IE 9 only (#12537)` as we won't be able to remove this code path anyway due to CSS vars.